app/src/main/java/us/shandian/giga/ui/fragment/MissionsFragment.java

```diff
@@ -208,6 +208,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
     }
 
+    public void areYouSureToDeleteAllFiles(){
```

Comment:
I personally don't like this name. It should be something along the lines of `showDeleteFileConfirmationAlert`. The current name is a string that you would show a user and does not really tell the developer whtat the method does